packages/grid/data-grid/src/DataGrid.test.tsx

```diff
@@ -32,6 +32,16 @@ describe('<DataGrid />', () => {
 
     // Adapation of describeConformance()
     describe('Material-UI component API', () => {
+      it(`resizable true should throw error`, () => {
+        defaultProps.columns[0].resizable = true;
+        expect(() =>
+          render(
+            <div style={{ width: 300, height: 300 }}>
+              <DataGrid {...defaultProps} />
+            </div>,
+          ),
+        ).toThrow();
```

Comment:
This is the Jest API

```suggestion
        ).to.throw();
```

Reply:
Can we assert the error message too? To make sure it's not unrelated.